packages/frontend/src/components/events/userEvents/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+import { Link } from "react-router-dom";
+import '../eventCard.css'
+
+type Props = {
+    repReceived: string;
+    attester: string;
+    epoch: string;
+}
+
+const UserEvent: FC<Props> = ({ repReceived, attester, epoch })  => {
```

Comment:
This is kinda hard to explain because we don't have a lot of data yet, but the idea behind the stores is that all data lives there and we don't pass things like `repReceived`, `epoch`, `attester` to components.

Instead each event (and every other object) has an identifier, and we load the object from the store using the identifier. That way we have a single source of truth, and we can change that source and change the ui in all places. Basically we get reactive elements for free and avoid state conflicts that arise by passing values around.

Then we access `repReceived`, `epoch`, `attester` on the object that we loaded from the store.